ee/src/main/java/org/jboss/as/ee/metadata/AbstractEEAnnotationProcessor.java

```diff
@@ -58,6 +58,12 @@ public final void deploy(final DeploymentPhaseContext phaseContext) throws Deplo
             for (Map.Entry<String, ClassAnnotationInformation<?, ?>> entry : data.entrySet()) {
                 EEModuleClassDescription clazz = eeModuleDescription.addOrGetLocalClassDescription(entry.getKey());
```

Comment:
`entry.getValue()` is used multiple times and I'd suggest introduce a variable to shorten the statements.

The same for `deploymentUnit.getParent()`